Formula/kubernetes-cli.rb

```diff
@@ -27,7 +27,21 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
+
     (bash_completion/"kubectl").write Utils.popen_read("#{bin}/kubectl completion bash")
+    (zsh_completion/"_kubectl").write Utils.popen_read("#{bin}/kubectl completion zsh")
+  end
+
+  def caveats; <<-EOS.undent
+    Add the following to ~/.zshrc to enable zsh completion:
+      source #{HOMEBREW_PREFIX}/share/zsh/site-functions/_kubectl
```

Comment:
This is a well-established convention for `zsh` completion files. A completion definition that doesn't get picked up by `compinit` is effectively broken. We should just add the `#compdef kubectl` line at the start of the file when we generate it, and remove the caveat.